src/utils/filter.ts

```diff
@@ -63,3 +71,21 @@ export const makeCampRoomHitFilter = ({
 
   return isWithinBounds(checkPercent, roomBounds);
 };
+
+export const filterUnreadPrivateChats = (
+  chats: WithId<PrivateChatMessage>[],
```

Comment:
It's probably a good habit to get into to make all arrays that are passed to things `readonly` in our types, to avoid/catch unintended mutations.

```suggestion
  chats: readonly WithId<PrivateChatMessage>[],
```

Reply:
Applied.